lib/teams_collection.rb

```diff
@@ -1,18 +1,42 @@
 require_relative './team'
-require 'CSV'
+require_relative './game_teams_collection'
```

Comment:
It's probably worth going through one more time and making sure we're all on the same page about when and where to use require or require_relative, I know I'm still struggling a little bit to fully wrap my mind around that